daemon/CryptoImpl/cryptopluginfunctionwrappers.cpp

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright (C) 2018 Jolla Ltd.
+ * Contact: Chris Adams <chris.adams@jollamobile.com>
+ * All rights reserved.
+ * BSD 3-Clause License, see LICENSE.
+ */
+
+#include "cryptopluginfunctionwrappers_p.h"
+#include "logging_p.h"
+
+using namespace Sailfish::Crypto;
+using namespace Sailfish::Crypto::Daemon::ApiImpl;
+
+/* These methods are to be called via QtConcurrent */
+
+bool CryptoPluginWrapper::isLocked(
+        CryptoPlugin *plugin)
```

Comment:
The parameter doesn't need to be on second line. Would expect const in such functions, but I guess that could be an issue for some plugins.